packages/astro/src/react/controlComponents.tsx

```diff
@@ -1,8 +1,9 @@
 import type { CheckAuthorizationWithCustomPermissions } from '@clerk/types';
 import type { PropsWithChildren } from 'react';
 
+import { $csrState } from '../stores/internal';
```

Comment:
Importing this from internal unless we decide to export the Clerk loading status as another store to user

Reply:
I actually think exporting the loading status (`$isLoadingClerkStore`) would help userland implement their own stuff and not do hacky stuff to check if Clerk loaded status

Reply:
in user land can't they do 

```tsx
import { $clerkStore } from '@clerk/astro/client';

const loadedClerk = () => {
  return new Promise((res) => {
    $clerkStore.subscribe((clerk) => {
      if(clerk.loaded) res(clerk)
    } )
  });
}
```

or


```tsx

 // outside the component
 $isLoadingClerkStore = computed($clerkStore,(clerk) => clerk.loaded)
 // inside the component
 const { isLoaded } = useStore($isLoadingClerkStore);

```

Reply:
Thanks. I remember we talked about this in ECO-4